src/40_Dynamic_AMP/Multi_Page_Flow.html

```diff
@@ -0,0 +1,537 @@
+  max-width: 420px;
+  margin: 1rem auto;
+}
+.stepper.sliding .page {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  width: 100%;
+  height: 100%;
+  padding: 16px;
+  transition: transform 0.25s cubic-bezier(0, 0, 0.3, 1);
+}
+.stepper.sliding .page.active {
+  transform: translateX(0%);
+  pointer-events: auto;
+}
+.stepper.sliding .page:not(.active) {
+  opacity: 0.5;
+  pointer-events: none;
+}
+.stepper.sliding .page.next {
+  transform: translateX(100%);
+}
+.stepper.sliding .page.previous {
+  transform: translateX(-100%);
+}
+.stepper.sliding .back {
+  position: relative;
+  height: 32px;
+  width: 100%;
+  max-width: 300px;
+  cursor: pointer;
+  line-height: 32px;
+  border: none;
+  background: none;
+  text-align: left;
+  padding: 0;
+  -webkit-appearance: none;
+}
+.stepper.sliding .back:after {
+  content: "";
+  position: absolute;
+  top: 10px;
+  right: 3px;
+  border: solid black;
+  border-width: 0 3px 3px 0;
+  display: inline-block;
+  padding: 3px;
+  transform: rotate(45deg);
+  -webkit-transform: rotate(45deg);
+}
+/* Survey styling */
+amp-selector.poll [option][selected] {
+  outline: none;
+}
+amp-selector.poll [option] {
+  display: flex;
+  align-items: center;
+}
+amp-selector.poll [option]:before {
+  transition: background 0.25s ease-in-out;
+  content: "";
+  display: inline-block;
+  width: 16px;
+  height: 16px;
+  margin: 8px;
+  border-radius: 100%;
+  border: solid 1px #b60845;
+  font-size: 14px;
+  line-height: 16px;
+}
+amp-selector.poll [option][selected]:before {
+  text-align: center;
+  content: "✓";
+  color: white;
+  background: #b60845;
+}
+amp-selector.poll [option][selected]:focus {
+  outline: -webkit-focus-ring-color auto 5px;
+}
+</style>
+</head>
+<body>
+  <h2 class="m0 p1 pt2">A simple mobile Stepper</h2>
+  <div class="p1">Use the browser back button to go to the previous view.</div>
+<!-- ## A simple Dialog -->
+<!-- We use an implicit state variable `currentPage` to keep track of the current view. Views are identified by numbers starting with `0` in their order of appearance. The view state is bound to each view using AMP's [`hidden` attribute](https://www.google.com/url?q=https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md%23-all-elements&sa=D&ust=1511453284339000&usg=AFQjCNGNPlF6oqBMqazKdfPiav-tPxaq9Q):
+
+```html
+<section [hidden]="currentPage > 0"> ... </section>
+```
+
+...and for initially hidden views we add the `hidden` attribute as a default:
+
+```html
+<section hidden [hidden]="currentPage != 1"> ... </section>
+```
+
+We update the `currentPage` variable to progress forward in the dialog. In this sample we're using `AMP.pushState(...)` instead of `AMP.setState(...)`. `AMP.pushState(...)` pushes a new entry onto the browser history stack, which allows the user to navigate back in the to use the browser's back button to move backwards in the dialog:
```

Comment:
add a link to the docs of `AMP.pushState`

Reply:
done